Documentation/astromd/all/CrossplayInput_8h.md

```diff
@@ -0,0 +1,125 @@
+---
+title: CrossplayInput.h File Reference
+---
+
+## Summary
+| Kind | View | Description |
+|------|------|-------------|
+|enum|[ERHAPI_CrossplayInput](/unreal-plugins/all/crossplayinput_8h/#CrossplayInput_8h_1a7bb2cd98a11e06372454b8e43954aee4)|Crossplay input preferences. Used to determine whether players can queue together, and the pools of players that are used to generate matches.|
+|RALLYHEREAPI_API FString|[EnumToString](/unreal-plugins/all/crossplayinput_8h/#CrossplayInput_8h_1acf8be298cc62c8682f577672142ff78b)(const ERHAPI_CrossplayInput & EnumValue)||
+|RALLYHEREAPI_API bool|[EnumFromString](/unreal-plugins/all/crossplayinput_8h/#CrossplayInput_8h_1a3a04ec8ddd6d5890029f7d74f629cc78)(const FString & EnumAsString, ERHAPI_CrossplayInput & EnumValue)||
+|RALLYHEREAPI_API FStringFormatArg|[ToStringFormatArg](/unreal-plugins/all/crossplayinput_8h/#CrossplayInput_8h_1a2a8ce1ec605faccfba992df818f04651)(const ERHAPI_CrossplayInput & Value)||
+|RALLYHEREAPI_API void|[WriteJsonValue](/unreal-plugins/all/crossplayinput_8h/#CrossplayInput_8h_1a2715cea3f86cd59779102789d52082da)(TSharedRef< TJsonWriter<>> & Writer, const ERHAPI_CrossplayInput & Value)||
+|RALLYHEREAPI_API bool|[TryGetJsonValue](/unreal-plugins/all/crossplayinput_8h/#CrossplayInput_8h_1afdfd6639dd6f0d3f58a989c8fbced512)(const TSharedPtr< FJsonValue > & JsonValue, ERHAPI_CrossplayInput & Value)||
+## Enums
+
+
+
+
+### `ERHAPI_CrossplayInput` <a id="CrossplayInput_8h_1a7bb2cd98a11e06372454b8e43954aee4"></a>
+Crossplay input preferences. Used to determine whether players can queue together, and the pools of players that are used to generate matches.
+
+
+
+| Enumerator | Initializer|
+|------------|------------|
+|`UMETA`|`=(ToolTip="No crossplay input preferences. Can play with players using any input")`|
```

Comment:
Looks like this processed the macro after the enum as the actual enum contents.  Not sure if this is a doxygen or converter issue.

Reply:
Looks like that comes straight from the Doxygen. The syntax those macros create has confused the hell out of it.

```
      <sectiondef kind="enum">
      <memberdef kind="enum" id="CrossplayInput_8h_1a7bb2cd98a11e06372454b8e43954aee4" prot="public" static="no" strong="yes">
        <type>uint8</type>
        <name>ERHAPI_CrossplayInput</name>
        <enumvalue id="CrossplayInput_8h_1a7bb2cd98a11e06372454b8e43954aee4aa3f80d8f0f1b4ffa8c475f5d60ea5635" prot="public">
          <name>UMETA</name>
          <initializer>=(ToolTip=&quot;No crossplay input preferences. Can play with players using any input&quot;)</initializer>
          <briefdescription>
          </briefdescription>
          <detaileddescription>
          </detaileddescription>
        </enumvalue>
        <enumvalue id="CrossplayInput_8h_1a7bb2cd98a11e06372454b8e43954aee4aa3f80d8f0f1b4ffa8c475f5d60ea5635" prot="public">
          <name>UMETA</name>
          <initializer>=(ToolTip=&quot;Can only play with players using a keyboard and mouse&quot;)</initializer>
          <briefdescription>
          </briefdescription>
          <detaileddescription>
          </detaileddescription>
        </enumvalue>
        <enumvalue id="CrossplayInput_8h_1a7bb2cd98a11e06372454b8e43954aee4aa3f80d8f0f1b4ffa8c475f5d60ea5635" prot="public">
          <name>UMETA</name>
          <initializer>=(ToolTip=&quot;Can only player with players using a gamepad/controller&quot;)</initializer>
          <briefdescription>
          </briefdescription>
          <detaileddescription>
          </detaileddescription>
        </enumvalue>
        <enumvalue id="CrossplayInput_8h_1a7bb2cd98a11e06372454b8e43954aee4aa3f80d8f0f1b4ffa8c475f5d60ea5635" prot="public">
          <name>UMETA</name>
          <initializer>=(ToolTip=&quot;Can only player with players using a touch screen&quot;)</initializer>
          <briefdescription>
          </briefdescription>
          <detaileddescription>
          </detaileddescription>
        </enumvalue>
        <briefdescription>
<para>Crossplay input preferences. Used to determine whether players can queue together, and the pools of players that are used to generate matches. </para>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="/home/asayman/UnrealDev/unreal-plugins/RallyHereAPI/Source/RallyHereAPI/Public/CrossplayInput.h" line="19" column="1" bodyfile="/home/asayman/UnrealDev/unreal-plugins/RallyHereAPI/Source/RallyHereAPI/Public/CrossplayInput.h" bodystart="20" bodyend="25"/>
      </memberdef>
```

Reply:
Is doxygen able to understand specific macros being noop and just ignoring it?